packages/app-desktop/gui/MainScreen/commands/showPrompt.ts

```diff
@@ -0,0 +1,41 @@
+import { CommandRuntime, CommandDeclaration, CommandContext } from '@joplin/lib/services/CommandService';
+
+export const declaration: CommandDeclaration = {
+	name: 'showPrompt',
+};
+
+enum PromptInputType {
+    Dropdown = 'dropdown',
+    Datetime = 'datetime',
+	Tags = 'tags',
+	Text = 'text',
+}
```

Comment:
Use tabs. Could you make sure you have the commit hook installed as it's quite annoying having to ask for something like this. Or maybe you made changes directly on GitHub without testing which is even worse.

Reply:
Done. That's weird, I don't know why this happening. Usually, the hook runs every time for me. I guess I committed while being in `packages/app-desktop` directory. Maybe that's why the hooks didn't run.